### ISSUE TITLE
Incorrect uptime of BGP neighbours 

### ISSUE THREAD

**Issue by stkonst** [alice-lg/alice-lg#31]
Hi team,

In our Alice deployment we see in the BGP sessions Established section an uptime of 292 years, both for the IPv4 version and the IPv6 version. You can observe it clearly in the following screenshot. 
![screenshot-alice2](https://user-images.githubusercontent.com/17049385/76330133-c29c0f00-62ed-11ea-800b-ea828cf6b39a.png)

Can you provide a fix for this issue? The birdwatcher is attached to a Bird v2 instance and we run the latest version both for Alice and birdwatcher. 

Thanks 
Stavros | AMS-IX NOC

**Comment by bodsink** [comment]
same issue with bird version 2

Thanks
Indri | BatamIX

**Comment by momorientes** [comment]
Same here with birdv2

Thanks
Moritz | Stuttgart-IX

**Comment by momorientes** [comment]
Hi,

this can be fixed in the alice config file, simply add this:
```
servertime = 2006-01-02T15:04:05.999999999Z07:00
servertime_short = 2006-01-02 15:04:05
servertime_ext = 2006-01-02 15:04:05
```
to the respective [source.rsfoo.birdwatcher] statement.

@mhannig: I guess you can resolve this issue however I'd suggest improving the documentation for the config file. I'd submit a PR but this was more or less trial and error

**Comment by mhannig** [comment]
I think we can add this to the default config

**Comment by mhannig** [comment]
btw is this bird2 specific?

**Comment by mhannig** [comment]
Or maybe we can add to the documentation some guidelines like
if you are using XY in birdZ, then use 
    servertime = ...
    ....

otherwise:
    servertime = ...

**Comment by momorientes** [comment]
Hi Annika,

I have no bird1 installation I could test this with, all the insights I can give are that I'm using bird2 and the following time format settings:
```
timeformat base         iso long;
timeformat log          iso long;
timeformat protocol     iso long;
timeformat route        iso long;
```

I think adding this to the documentation as bird2 specific sounds good to me.

**Comment by stkonst** [comment]
Hi again and thank's for the reply. I modified my alice.conf file and now I have this 
`
[source.rsfoo.birdwatcher]
# Optional: Examples for time format
# Please see https://golang.org/pkg/time/#pkg-constants for an
# explanation on how time parsing in go works.
servertime = 2006-01-02T15:04:05.999999999Z07:00
servertime_short = 2006-01-02 15:04:05
servertime_ext = 2006-01-02 15:04:05
`

I restarted the Alice process but the problem remains. Do I miss anything? 


Thank's again.

**Comment by stkonst** [comment]
Sorry, just realized my mistake and now I figured out where shall I place the workaround in the config. I did it and now I get the correct uptime being displayed.

**Comment by mwnmwn** [comment]
> Sorry, just realized my mistake and now I figured out where shall I place the workaround in the config. I did it and now I get the correct uptime being displayed.

hello could you share where to put this on ?  I'm trying to put in alice.conf but nothing happen

**Comment by SalehCoder** [comment]
> Hi,
> 
> this can be fixed in the alice config file, simply add this:
> 
> ```
> servertime = 2006-01-02T15:04:05.999999999Z07:00
> servertime_short = 2006-01-02 15:04:05
> servertime_ext = 2006-01-02 15:04:05
> ```
> 
> to the respective [source.rsfoo.birdwatcher] statement.
> 
> @mhannig: I guess you can resolve this issue however I'd suggest improving the documentation for the config file. I'd submit a PR but this was more or less trial and error

That fix worked for Bird 1.6.8.
